packages/gatsby/index.d.ts

```diff
@@ -17,6 +17,8 @@ import { GraphQLOutputType } from "graphql"
 import { PluginOptionsSchemaJoi, ObjectSchema } from "gatsby-plugin-utils"
 import { IncomingMessage, ServerResponse } from "http"
 
+export type AvailableFeatures = never // "image-service"
```

Comment:
A TODO or other note here would be helpful, I was unsure what this comment meant when I saw it

Reply:
I have some other PRs lined up so that's why I didn't add a TODO, this will be uncommented in the next one